code/datums/components/deployable.dm

```diff
@@ -41,39 +48,42 @@
 /datum/component/deployable/proc/examine(datum/source, mob/user, list/examine_list)
 	SIGNAL_HANDLER
 
-	examine_list += span_notice("[source.p_they()] look[source.p_s()] like [source.p_they()] can be deployed into \a [deployed_name].")
+	examine_list += span_notice("It can be used <b>in hand</b> to deploy into \a [deployed_name].")
```

Comment:
Another very minor thing, but if `multiple_deployments` makes an item more of a "thing dispenser" the word into maybe doesn't make sense there? 
Might want it to be `deploy a [thing]` or `deploy into a [thing]` based on the properties.